mods/ra/rules/player.yaml

```diff
@@ -18,6 +18,7 @@ Player:
 		LowPowerSlowdown: 3
 		QueuedAudio: Building
 		SpeedUp: True
+		BuildTimeSpeedReduction: 100, 75, 60, 50
```

Comment:
The queue build menu tabs would be the place to display that but that would likely make the tooltip look bloated. Except for the unit icon tooltips the game has no way of displaying specific values to the player.